conda_build/environ.py

```diff
@@ -148,16 +148,17 @@ def get_dict(m=None, prefix=None):
         except NotImplementedError:
             d['CPU_COUNT'] = "1"
 
-    if m.get_value('source/git_url'):
-        git_url = m.get_value('source/git_url')
-        if '://' not in git_url:
-            # If git_url is a relative path instead of a url, convert it to an abspath
-            if not isabs(git_url):
-                git_url = join(m.path, git_url)
-            git_url = normpath(join(m.path, git_url))
-        d.update(**get_git_build_info(d['SRC_DIR'],
-                                      git_url,
-                                      m.get_value('source/git_rev')))
+    if m:
+        if m.get_value('source/git_url'):
```

Comment:
This is super nit-picky, but wouldn't it be cleaner to combine this into a single line?  (That turns this into a 1-line PR.)

```python
if m and m.get_value('source/git_url'):
```